tests/containers/podman_bci_systemd.pm

```diff
@@ -23,6 +23,9 @@ sub run {
     record_info('Test', 'Launch a container with systemd');
     assert_script_run("podman run -d -p 80:80 --name nginx $image");
 
+    # Wait for the container to be up
+    script_retry("podman exec nginx cat /etc/os-release", retry => 10, delay => 15);
```

Comment:
Waiting for the port to be open could be quicker or maybe also waiting for a line with `podman logs`:

```suggestion
    script_run("until (echo >/dev/tcp/127.0.0.1/80) >/dev/null 2>&1 ; do sleep 1 ; done", timeout => 150);
```